### ISSUE TITLE
Request for new ontology [Exercise Medicine Ontology]

### ISSUE THREAD

**Issue by DarkKnight0-0** [OBOFoundry/OBOFoundry.github.io#2615]
### Title

Exercise Medicine Ontology

### Short Description

A core reference ontology built upon BFO about exercise medicine and it contains the related terms for healthy people, people with chronic conditions and people living with diability to exercise.

### Description

The concept of "exercise is medicine" is gaining traction globally, highlighting the importance of personalized exercise prescriptions for better efficacy than standardized approaches.  However, current guidelines often need more support for individualized prescriptions, posing a significant challenge. To bridge this gap, we gathered data from established guidelines, databases, and articles to develop the Exercise Medicine Ontology (EXMO), intending to offer comprehensive support for personalized exercise prescriptions. EXMO encompasses physical activity terms, health status terms, exercise prescription terms, and other related concepts. It has successfully undergone expert evaluation and consistency validation using the ELK and JFact reasoners.  EXMO has the potential to provide a much-needed standard for individualized exercise prescription. Beyond prescription standardization, EXMO can also be an excellent tool for supporting databases and recommendation systems. In the future, it could serve as a valuable reference for the development of sub-ontologies and could facilitate the formation of an ontology network.

### Identifier Space

EXMO

### License

CC-BY 4.0

### Domain

health

### Source Code Repository

https://github.com/DarkKnight0-0/exmo

### Homepage

https://github.com/DarkKnight0-0/exmo

### Issue Tracker

https://github.com/DarkKnight0-0/exmo/issues

### Contribution Guidelines

https://github.com/DarkKnight0-0/exmo/blob/master/CONTRIBUTING.md

### Ontology Download Link

https://github.com/DarkKnight0-0/exmo/blob/master/exmo.owl

### Contact Name

Xingyun Liu

### Contact Email

xyz19940216@163.com

### Contact GitHub Username

 DarkKnight0-0

### Contact ORCID Identifier

0000-0002-9295-2767

### Formats

- [X] OWL RDF/XML (.owl)
- [X] OBO (.obo)
- [ ] OBO Graph JSON (.json)

### Dependencies

- bfo
- ro
- doid
- hsapdv

### OBO Foundry Pre-registration Checklist

- [X] I have read and understood [the registration process instructions](http://obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.html) and the [registration checklist](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/docs/RegistrationChecklist.md).
- [X] There is no other ontology in the OBO Foundry which would be an appropriate place for my terms. If there were, I have contacted the editors, and we decided in mutual agreement that a separate ontology is more appropriate.
- [X] My ontology has a specific release file with a version IRI and a `dc:license` annotation, serialised in RDF/XML.
- [X] My identifiers (classes and properties IRIs) are formatted according to the [OBO Foundry Identifier Policy](http://obofoundry.org/id-policy)
- [X] My term labels are in English and conform to the OBO Foundry [Naming Conventions](http://obofoundry.org/http://obofoundry.org/principles/fp-012-naming-conventions.html)
- [X] I understand that term definitions are key to understanding the intentions of a term, especially when the ontology is used in curation. I made sure that a reasonable majority of terms in my ontology--and all top level terms--have definitions, in English, using the [IAO:0000115](https://ontobee.org/ontology/IAO?iri=http://purl.obolibrary.org/obo/IAO_0000115) property.
- [X] For every term in my ontology, I checked whether another OBO Foundry ontology has one with the same meaning. If so, I re-used that term directly (not by cross-reference, by directly using the IRI).
- [X] For all relationship properties (Object and Data Property), I checked whether [the Relation Ontology (RO)](https://ontobee.org/ontology/catalog/RO?iri=http://www.w3.org/2002/07/owl%23ObjectProperty) includes an appropriate one. I understand that aligning with RO is an essential part of the overall alignment between OBO ontologies!
- [X] For the selection of appropriate annotation properties, I looked at [OMO](https://ontobee.org/ontology/catalog/OMO?iri=http://www.w3.org/2002/07/owl%23AnnotationProperty) first. I understand that aligning ontology metadata and term-level metadata is essential for cross-integration of OBO ontologies.
- [X] If I was not sure about the meaning of any of the checkboxes above, I have consulted with a member of the OBO Foundry for advice, e.g., through the [obo-discuss](https://groups.google.com/g/obo-discuss) Google Group.
- [X] The requested ID space does not conflict with another ID space found in other registries such as the Bioregistry and BioPortal, see [here](https://obofoundry.org/id-policy) for a complete list.

**Comment by ddooley** [comment]
Thank you for this request.  We are adding your ontology to the OBO Dashboard as a first step, and will take further action - feedback or assigning a reviewer if it passes, shortly after. We're sending an email about this to you too.

**Comment by anitacaron** [comment]
Hi @DarkKnight0-0 You can check the OBO NOR Dashboard results [here](https://obofoundry.org/obo-nor.github.io/dashboard/exmo/dashboard.html)

**Comment by anitacaron** [comment]
@DarkKnight0-0 Please note that there's another step that runs a lexical matching tool to check for lexical overlap with existing OBO ontologies. The results will be available soon after 22 July.

**Comment by DarkKnight0-0** [comment]
> * [OBO NOR Dashboard](https://obofoundry.org/obo-nor.github.io/dashboard/index.html).

Thank you for the review. We have fixed the license problem.

**Comment by pfabry** [comment]
@DarkKnight0-0 
The lexical matching hasn't find any duplicate in your ontology.

**Comment by shawntanzk** [comment]
Hi, just noticed that you used ODK (from this [readme](https://github.com/DarkKnight0-0/exmo/blob/master/src/ontology/README-editors.md) and the look of your repo), however, you do declare a lot of terms from other ontologies directly in your edit file rather than as an import. This is not directly a problem, however, according to [principal 16 on maintenance](https://obofoundry.org/principles/fp-016-maintenance.html), I would like to know how you plan to keep these terms up to date with the ontologies as they update. 

PS I would also recommend using the ODK inbuilt dynamic import system which automatically does this. Hope that helps :)

**Comment by pfabry** [comment]
@jsstevenson has been assigned to review this ontology.

**Comment by DarkKnight0-0** [comment]
> Hi, just noticed that you used ODK (from this [readme](https://github.com/DarkKnight0-0/exmo/blob/master/src/ontology/README-editors.md) and the look of your repo), however, you do declare a lot of terms from other ontologies directly in your edit file rather than as an import. This is not directly a problem, however, according to [principal 16 on maintenance](https://obofoundry.org/principles/fp-016-maintenance.html), I would like to know how you plan to keep these terms up to date with the ontologies as they update.
> 
> PS I would also recommend using the ODK inbuilt dynamic import system which automatically does this. Hope that helps :)

Thanks for the advice. We will use the dynamic import system to update the next version.

**Comment by jsstevenson** [comment]
Hi @DarkKnight0-0 -- thanks again for your submission. Below is my initial review. You may use this issue thread to ask for clarification and/or to update us on progress. Please feel free to voice any other questions or concerns as well.

# Criteria

## 1. Ontology scope

* *Was the ontology developed for a very specific purpose or community?*: Yes. As noted in the request PR, exercise medicine is a specific field with a corresponding academic community.

* *Do the terms fall within the ontology’s stated target domain of knowledge?*: As discussed below, there are many terms that are out of scope, and should be imported from higher-level and domain-specific ontologies -- but there is a core set of terms that are reasonable to define here.

## 2. Terms with the new ontology prefix

* *Do the terms follow the OBO identifier scheme?*: No, URIs do not consistently conform to the [OBO Foundry Identifier Policy](http://obofoundry.org/id-policy) -- I'm seeing a few different URI schemas:

  * `http://purl.obolibrary.org/obo/emo.owl/EXM_0000001`
  * `http://purl.obolibrary.org/obo/exmo.owl#patient`
  * `http://purl.obolibrary.org/obo/exmo.owl/EXM_0000214`

These must all be in the format of e.g. `http://purl.obolibrary.org/obo/EXMO_0000001`.

* *Are there terms with the same meaning available in another OBO Foundry ontology?*: I did not perform an exhaustive check, but upon skimming the ontology I found a number of classes that are defined elsewhere in OBO domain ontologies:

  * `http://purl.obolibrary.org/obo/exmo.owl/EXM_0000039` "skeletal muscle": `http://purl.obolibrary.org/obo/UBERON_0001134`
  * `http://purl.obolibrary.org/obo/exmo.owl/EXM_0000043` "basal metabolic rate": `http://purl.obolibrary.org/obo/CMO_0003955`
  * `http://purl.obolibrary.org/obo/exmo.owl/EXM_0000056` "total lung capacity": `http://purl.obolibrary.org/obo/CMO_0000380`
  * `http://purl.obolibrary.org/obo/exmo.owl/EXM_0000074` "core body temperature": `http://purl.obolibrary.org/obo/CMO_0001036`
  * ... many other possible terms of overlap with CMO classes -- pulse, blood pressure, heart rate, cholesterol level, etc
  * `http://purl.obolibrary.org/obo/exmo.owl/EXM_0000097` "questionnaire": `http://purl.obolibrary.org/obo/OBI_0001000`
  * `http://purl.obolibrary.org/obo/exmo.owl/EXM_0000368` "electromyography": `http://purl.obolibrary.org/obo/MAXO_0035091`
  * `http://purl.obolibrary.org/obo/exmo.owl#patient` "patient": several possible candidates
  * `http://purl.obolibrary.org/obo/exmo.owl/EXM_0000214` "person": several possible candidates

I suspect there are several other cases as well. New OBO ontologies are required to only define new terms unique to their intended scope, and import terms from existing ontologies when available. If a term from another ontology is within its scope but its definition is somehow lacking, we would strongly prefer that you work with maintainers of that ontology to improve their definition, rather than creating a new, redundant term.

* *Is there another OBO Foundry ontology whose scope covers any of the new terms?*: Noted above that there's a good deal of possible overlap with terms already defined in Uberon, CMO, the Cell Ontology, and others. There are definitely a decent number of terms in here that wouldn't be within scope anywhere else (e.g. exercise equipment).

* I found a handful of cases where term definitions were copied directly from other resources, without attribution. For example, the textual definition for `EXM_0000378` ("glutamate") is from NCIt, the definition for `EXM_0000373` ("endorphin") is from MeSH (at minimum this definition should probably be revised given the different context), and the definition for `EXM_0000375` ("norepinephrine") is from PubChem. Each of these example terms should probably be imported from another OBO ontology anyway, but this does raise independent concerns about attribution (both in terms of good academic practice and FAIRness, and for potential data license issues). It is very important that direct textual content be properly annotated to reflect its original source.

* Around 80 of the 380 new classes lack definitions. I think that's within acceptable limits but it might be good to review and provide definitions for more ambiguous instances.

## 3. Correct use of imported terms

* *If the ontology reuses terms from other OBO ontologies, are they used accurately?*: There's reasonable use of DO, uberon, and human developmental stage classes. As noted by @shawntanzk, they're manually declared rather than imported via ODK -- so there could be some slippage in the future unless this is remedied.

* *Are imported terms in appropriate hierarchies, and do they preserve the term’s upper-level alignment?*:

* *Are any additional axioms used for these terms correct in both a technical (e.g. passes reasoning) and substantive sense?*: I'm concerned with how restrictions are added to some upper ontologic terms. For example, this restriction that `BFO_0000019` ("quality") must be a quality of a `EXM_0000214` ("person").

```xml
<owl:Class rdf:about="http://purl.obolibrary.org/obo/BFO_0000019">
    <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/BFO_0000020"/>
    <rdfs:subClassOf>
        <owl:Restriction>
            <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/RO_0000080"/>
            <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/exmo.owl/EXM_0000214"/>
        </owl:Restriction>
    </rdfs:subClassOf>
```

Similar restrictions are added to `BFO_0000034` and `HsapDv_0000000`. It's not clear why it would be preferable to further restrict the BFO "quality" concept rather than creating some kind of new class reflecting the particular type of quality that is associated with a person. Because this is a fairly significant case, I think it's reasonable to require that this is fixed, assuming there isn't a pressing justification.

## 4. Basic review of axiomatic patterns

* *Are axioms generally stated simply or are they highly complex? (Highly complex axioms will require extra scrutiny.)*: Generally quite straightforward. No complaints.

* *Are existential restrictions used correctly?*: yes.

## 5. Appropriate use of object properties

* *Are object properties used in a manner consistent with their definitions, domain, and range?*:

  * There's one new object property defined, as far as I can tell  -- `EXM_0000006`, "Abbreviation". I'm not sure if it would be desirable to instead import this from somewhere, given that this is obviously not the first place that you'd want to annotate a class with an abbreviation, but I'm not sure if there's a standard source to pull from.

  * a lot of the values for `oboInOwl:hasDbXref` are formatted in a nonstandard way, as `<Name>: <LUI>` (for example `UMLS CUI: C0005938`) rather than a more standard CURIE. These should be restructured to ensure they are more easily computable by removing the space and using prefixes as defined in a central entity like the [Bioregistry](https://bioregistry.io/) or [identifiers.org](https://identifiers.org). So, for example, I would expect this value to be `umls:C0005938`.

## 6. Responsiveness to suggested changes

* *Have the developers been willing to fix any identified issues during the review?*: The developers have been responsive so far to individual comments. This is the first iteration of the formal review.

## Other

* The EXMO repository is located under a GitHub user account -- we generally recommend that ontologies are housed within a GitHub organization to ensure sustainability.

# Action items

- [ ] **MUST**: Fix URI format for all new terms to adhere to the OBO identifier standard
- [ ] **MUST**: import all out-of-scope classes from other OBO ontologies, and not define redundantly within EXMO. At a glance, I found a number of violations; I would recommend a term-by-term review.
- [ ] **MUST**: annotate all cases of textual definitions copied from external sources appropriately. I also found a number of instances of this, and would recommend a term-by-term review.
- [ ] **CONSIDER**: Review remaining terms lacking textual definitions and add where helpful
- [ ] **SHOULD**: Import terms from other ontologies via ODK rather than re-declaring manually
- [ ] **MUST**: remove axioms/restrictions placed on imported terms, and develop alternate constructions to perform the equivalent logical operations.
- [ ] **MUST**: format `hasDbXref` property values as proper identifiers and using standardized namespace prefixes
- [ ] **CONSIDER**: relocate the EXMO repository to a GitHub organization, rather than an individual user account's repo

**Comment by wdduncan** [comment]
Hello. We are checking in about the status of the requested update to your ontology. Please let us know if you have further questions.

**Comment by DarkKnight0-0** [comment]
> Hello. We are checking in about the status of the requested update to your ontology. Please let us know if you have further questions.

Thanks for your comments, the criteria is very clear and easy to understand. We will revise the problems and release a new version.

**Comment by nicolevasilevsky** [comment]
@DarkKnight0-0 Hi - thanks for your comments, if you could kindly provide an update on the status of your release, we would appreciate it, thanks!

**Comment by jsstevenson** [comment]
Due to inactivity for the past two months, this issue will be closed. When ready, you are welcome to re-open the issue, at which point we will be glad to continue the process. Thanks for your interest in submitting your ontology to the OBO Foundry.

**Comment by Exercise-Medicine-Ontology** [comment]
We apologize for the delay and have resolved the issues. The new address is below.

Source Code Repository
https://github.com/Exercise-Medicine-Ontology/EXMO

Homepage
https://github.com/Exercise-Medicine-Ontology/EXMO

Issue Tracker
https://github.com/Exercise-Medicine-Ontology/EXMO/issues

Contribution Guidelines
https://github.com/Exercise-Medicine-Ontology/EXMO/blob/master/CONTRIBUTING.md

Ontology Download Link
https://github.com/Exercise-Medicine-Ontology/EXMO/blob/master/exmo.owl

**Comment by pfabry** [comment]
Resubmission of EXMO

**Comment by pfabry** [comment]
The ontology is available on the [NOR Dashboard](https://obofoundry.org/obo-nor.github.io/dashboard/index.html).
The license annotation in the ontology file need to be corrected. It must indicate only the license type. ex. `<dcterms:license rdf:resource="https://creativecommons.org/licenses/by/4.0/"/>`
There are still some duplicate terms, cf, lexical matching terms file attached.

**Comment by Exercise-Medicine-Ontology** [comment]
> [EXMO lexmatch 20250415.txt](https://github.com/user-attachments/files/19761002/EXMO.lexmatch.20250415.txt)
> 
> The ontology is available on the [NOR Dashboard](https://obofoundry.org/obo-nor.github.io/dashboard/index.html). The license annotation in the ontology file need to be corrected. It must indicate only the license type. ex. `<dcterms:license rdf:resource="https://creativecommons.org/licenses/by/4.0/"/>` There are still some duplicate terms, cf, lexical matching terms file attached.

Thank you for the comments. We will fix it and release a new version soon.

**Comment by Exercise-Medicine-Ontology** [comment]
> [EXMO lexmatch 20250415.txt](https://github.com/user-attachments/files/19761002/EXMO.lexmatch.20250415.txt)
> 
> The ontology is available on the [NOR Dashboard](https://obofoundry.org/obo-nor.github.io/dashboard/index.html). The license annotation in the ontology file need to be corrected. It must indicate only the license type. ex. `<dcterms:license rdf:resource="https://creativecommons.org/licenses/by/4.0/"/>` There are still some duplicate terms, cf, lexical matching terms file attached.

We have fixed it and upload the new version to Github.
https://github.com/Exercise-Medicine-Ontology/EXMO

**Comment by sebastianduesing** [comment]
Hello @Exercise-Medicine-Ontology! I've been asked to take over for the next round of review. Thanks for following up with the new version—it's clear that you've put in quite a bit of work to address the issues raised in the previous review. My review is below. I will discuss this with the rest of the OBO Operations Committee at our next call on 2025-05-13. Please feel free to respond with questions, comments, or updates.

# Criteria
## Ontology scope
* _Was the ontology developed for a very specific purpose or community?_ Yes.
* _Do the terms fall within the ontology’s stated target domain of knowledge?_ As James said, the core set of terms is reasonably contained within the domain of exercise medicine, and it appears that substantial effort has been made to replace out-of-scope terms with terms from other OBO ontologies. However, there remain a handful of terms with EXMO IRIs that should likely be sourced from other ontologies, as I discuss in more detail below.

## Terms with the new ontology prefix
* _Do the terms follow the OBO identifier scheme?_ Yes. The variance in identifier styles that James identified in the previous version has been corrected, and all IRIs now follow the `http://purl.obolibrary.org/obo/EXMO_0000001` style.
* _Are there terms with the same meaning available in another OBO Foundry ontology?_ The terms James identified in the prior review have been replaced with imports from the appropriate ontologies. However, there are still a handful of terms that may be able to be replaced with imports, including the following:
  * `EXMO:0000394` "caring for children" --> `GSSO:003899` "child care"
  * `EXMO:0000344` "recreational activity" --> `GSSO:008528` "recreation"
  * `EXMO:0000102` "body composition testing" --> `EFO:0005106` "body composition measurement"
* _Is there another OBO Foundry ontology whose scope covers any of the new terms?_ Yes. While the number of out-of-scope terms has been substantially reduced since the last review, there are still a few categories of terms that would benefit from being moved to other domain ontologies. In particular, I recommend the following:
  * Clinical measurement subclasses (e.g.,  `EXMO:0000073` "axillary temperature", `EXMO:0000075` "oral temperature", `EXMO:0000076` "rectal temperature", `EXMO:0000075` "tympanic temperature") should be in CMO.
  * Injury terms (e.g., `EXMO:0000364` "acute sprain of ankle joint" and `EXMO:0000363` "injury of cruciate ligament of knee") should be in DOID, a controlled vocabulary of diagnosis codes, or similar.
  * Demographic terms (`EXMO:0000284` "postpartum woman" and `EXMO:0000317` "perimenopausal woman") should be elsewhere. I'm not sure which ontologies would be best for these—perhaps GSSO? Terms like `OBI:0002413` "perimenopausal status" may also be relevant in relocating these terms.

## Correct use of imported terms
* _If the ontology reuses terms from other OBO ontologies, are they used accurately?_ Yes, as far as I can tell. Shawn and James previously noted that it would be good to use ODK for EXMO's imports for long-term stability; I'm not familiar enough with ODK to be certain whether EXMO has taken sufficient action to address Shawn and James' concerns, so I welcome their input on this.
* _Are imported terms in appropriate hierarchies, and do they preserve the term’s upper-level alignment?_ Yes, EXMO adheres to BFO for its own classes and its imports, and those imports are in appropriate hierarchies. I would recommend one change: currently, the CMO "clinical measurement" branch is imported under "quality". Strictly speaking, this is not in conflict with CMO's modeling, as CMO contains no terms above "clinical measurement", but CMO defines "clinical measurement" as "A quantitative or qualitative value which is the result of an act of assessing a morphological or physiological state or property in a single individual or sample or a group of individuals or samples, based on direct observation or experimental manipulation." Accordingly, it seems that CMO intends "clinical measurement" to be used for _values_ resulting from the process of measuring a characteristic, not the process or the characteristic, and thus it is probably better to import it as an information content entity, perhaps as a subclass of `IAO:0000109` "measurement datum". Clearer modeling in CMO itself would be helpful here.
* _Are any additional axioms used for these terms correct in both a technical (e.g. passes reasoning) and substantive sense?_ Yes. It appears that the issue with the addition of restrictions to upper-level terms like `BFO:0000019` ("quality") has been fixed by removing those restrictions as James recommended.

## Basic review of axiomatic patterns
* _Are axioms generally stated simply or are they highly complex? (Highly complex axioms will require extra scrutiny.)_ Yes, the axioms are generally quite simple and straightforward.
* _Are existential restrictions used correctly? (Typical mistakes include “R some (A and B and C)” to mean “(R some A and R some B and R some C)”)._ Yes.

## Appropriate use of object properties
* _Are object properties used in a manner consistent with their definitions, domain, and range? (Examples of incorrect usage include those based on some interpretation of the label of the object property but not actually fitting the property definition or domain and range.)_ Yes. The issue raised by James about the object property `EXMO:0000006` "abbreviation" being out of EXMO's scope has been resolved by removing that term. The issue of nonstandard values for `oboInOwl:hasDbXref` appears to have been resolved by standardizing values into CURIE format as recommended.

## Responsiveness to suggested changes
* Have the developers been willing to fix any identified issues during the review?_ Yes, it is clear that the EXMO team has made a substantial and broadly successful effort to address the issues raised in the previous round of review.

## Other
* It appears that some terms for exercise equipment have axioms relating them to their associated sports or games, while similar terms lack those axioms. For example, `EXMO:0000244` "golf club" has the axiom `'participates in' some 'golf'`, while `EXMO:0000243` "golf ball" has no such axiom. I recommend reviewing the "exercise equipment" branch to ensure that, if you intend to include axioms linking equipment items to sports/games, those axioms are applied consistently across all relevant terms in the branch.
* As previously noted, ~80 of the ~220 terms with EXMO IDs lack textual definitions. Prioritizing the terms closest to the core of EXMO's domain (e.g., `EXMO:0000350` "anaerobic exercise", `EXMO:0000379` "exercise plan", and similar) for receiving definitions would maximize the payoff of any further work done to add textual definitions.

# Action Items
Most of the previous list of action items created by James have been successfully addressed by EXMO. The following list includes the remaining items from that list plus newly-identified items.

- [ ] __MUST:__ Review terms to identify remaining out-of-scope classes and ensure that out-of-scope classes are imported from other ontologies if equivalents already exist elsewhere OR request them from ontologies with appropriate domains if not.
- [ ] __SHOULD:__ Import `CMO:0000000` "clinical measurement" as a subclass of `IAO:0000109` "measurement datum" rather than as a subclass of "quality".
- [ ] __SHOULD__: Review axioms in general, but particularly in the "exercise equipment" branch, to ensure desired axioms are applied consistently across similar terms.
- [ ] __SHOULD__: Add textual definitions, prioritizing terms for core domain concepts, like `EXMO:0000350` "anaerobic exercise".

**Comment by Exercise-Medicine-Ontology** [comment]
> Hello [@Exercise-Medicine-Ontology](https://github.com/Exercise-Medicine-Ontology)! I've been asked to take over for the next round of review. Thanks for following up with the new version—it's clear that you've put in quite a bit of work to address the issues raised in the previous review. My review is below. I will discuss this with the rest of the OBO Operations Committee at our next call on 2025-05-13. Please feel free to respond with questions, comments, or updates.
> 
> # Criteria
> ## Ontology scope
> * _Was the ontology developed for a very specific purpose or community?_ Yes.
> * _Do the terms fall within the ontology’s stated target domain of knowledge?_ As James said, the core set of terms is reasonably contained within the domain of exercise medicine, and it appears that substantial effort has been made to replace out-of-scope terms with terms from other OBO ontologies. However, there remain a handful of terms with EXMO IRIs that should likely be sourced from other ontologies, as I discuss in more detail below.
> 
> ## Terms with the new ontology prefix
> * _Do the terms follow the OBO identifier scheme?_ Yes. The variance in identifier styles that James identified in the previous version has been corrected, and all IRIs now follow the `http://purl.obolibrary.org/obo/EXMO_0000001` style.
> * _Are there terms with the same meaning available in another OBO Foundry ontology?_ The terms James identified in the prior review have been replaced with imports from the appropriate ontologies. However, there are still a handful of terms that may be able to be replaced with imports, including the following:
>   
>   * `EXMO:0000394` "caring for children" --> `GSSO:003899` "child care"
>   * `EXMO:0000344` "recreational activity" --> `GSSO:008528` "recreation"
>   * `EXMO:0000102` "body composition testing" --> `EFO:0005106` "body composition measurement"
> * _Is there another OBO Foundry ontology whose scope covers any of the new terms?_ Yes. While the number of out-of-scope terms has been substantially reduced since the last review, there are still a few categories of terms that would benefit from being moved to other domain ontologies. In particular, I recommend the following:
>   
>   * Clinical measurement subclasses (e.g.,  `EXMO:0000073` "axillary temperature", `EXMO:0000075` "oral temperature", `EXMO:0000076` "rectal temperature", `EXMO:0000075` "tympanic temperature") should be in CMO.
>   * Injury terms (e.g., `EXMO:0000364` "acute sprain of ankle joint" and `EXMO:0000363` "injury of cruciate ligament of knee") should be in DOID, a controlled vocabulary of diagnosis codes, or similar.
>   * Demographic terms (`EXMO:0000284` "postpartum woman" and `EXMO:0000317` "perimenopausal woman") should be elsewhere. I'm not sure which ontologies would be best for these—perhaps GSSO? Terms like `OBI:0002413` "perimenopausal status" may also be relevant in relocating these terms.
> 
> ## Correct use of imported terms
> * _If the ontology reuses terms from other OBO ontologies, are they used accurately?_ Yes, as far as I can tell. Shawn and James previously noted that it would be good to use ODK for EXMO's imports for long-term stability; I'm not familiar enough with ODK to be certain whether EXMO has taken sufficient action to address Shawn and James' concerns, so I welcome their input on this.
> * _Are imported terms in appropriate hierarchies, and do they preserve the term’s upper-level alignment?_ Yes, EXMO adheres to BFO for its own classes and its imports, and those imports are in appropriate hierarchies. I would recommend one change: currently, the CMO "clinical measurement" branch is imported under "quality". Strictly speaking, this is not in conflict with CMO's modeling, as CMO contains no terms above "clinical measurement", but CMO defines "clinical measurement" as "A quantitative or qualitative value which is the result of an act of assessing a morphological or physiological state or property in a single individual or sample or a group of individuals or samples, based on direct observation or experimental manipulation." Accordingly, it seems that CMO intends "clinical measurement" to be used for _values_ resulting from the process of measuring a characteristic, not the process or the characteristic, and thus it is probably better to import it as an information content entity, perhaps as a subclass of `IAO:0000109` "measurement datum". Clearer modeling in CMO itself would be helpful here.
> * _Are any additional axioms used for these terms correct in both a technical (e.g. passes reasoning) and substantive sense?_ Yes. It appears that the issue with the addition of restrictions to upper-level terms like `BFO:0000019` ("quality") has been fixed by removing those restrictions as James recommended.
> 
> ## Basic review of axiomatic patterns
> * _Are axioms generally stated simply or are they highly complex? (Highly complex axioms will require extra scrutiny.)_ Yes, the axioms are generally quite simple and straightforward.
> * _Are existential restrictions used correctly? (Typical mistakes include “R some (A and B and C)” to mean “(R some A and R some B and R some C)”)._ Yes.
> 
> ## Appropriate use of object properties
> * _Are object properties used in a manner consistent with their definitions, domain, and range? (Examples of incorrect usage include those based on some interpretation of the label of the object property but not actually fitting the property definition or domain and range.)_ Yes. The issue raised by James about the object property `EXMO:0000006` "abbreviation" being out of EXMO's scope has been resolved by removing that term. The issue of nonstandard values for `oboInOwl:hasDbXref` appears to have been resolved by standardizing values into CURIE format as recommended.
> 
> ## Responsiveness to suggested changes
> * Have the developers been willing to fix any identified issues during the review?_ Yes, it is clear that the EXMO team has made a substantial and broadly successful effort to address the issues raised in the previous round of review.
> 
> ## Other
> * It appears that some terms for exercise equipment have axioms relating them to their associated sports or games, while similar terms lack those axioms. For example, `EXMO:0000244` "golf club" has the axiom `'participates in' some 'golf'`, while `EXMO:0000243` "golf ball" has no such axiom. I recommend reviewing the "exercise equipment" branch to ensure that, if you intend to include axioms linking equipment items to sports/games, those axioms are applied consistently across all relevant terms in the branch.
> * As previously noted, ~80 of the ~220 terms with EXMO IDs lack textual definitions. Prioritizing the terms closest to the core of EXMO's domain (e.g., `EXMO:0000350` "anaerobic exercise", `EXMO:0000379` "exercise plan", and similar) for receiving definitions would maximize the payoff of any further work done to add textual definitions.
> 
> # Action Items
> Most of the previous list of action items created by James have been successfully addressed by EXMO. The following list includes the remaining items from that list plus newly-identified items.
> 
> * [ ]  **MUST:** Review terms to identify remaining out-of-scope classes and ensure that out-of-scope classes are imported from other ontologies if equivalents already exist elsewhere OR request them from ontologies with appropriate domains if not.[ ]  **SHOULD:** Import `CMO:0000000` "clinical measurement" as a subclass of `IAO:0000109` "measurement datum" rather than as a subclass of "quality".[ ]  **SHOULD**: Review axioms in general, but particularly in the "exercise equipment" branch, to ensure desired axioms are applied consistently across similar terms.[ ]  **SHOULD**: Add textual definitions, prioritizing terms for core domain concepts, like `EXMO:0000350` "anaerobic exercise".

Thanks for the review. We will fix these issuses and release a new version.
For the out-of-scope classes, before the speisific domain ontologies added the classes, should we keep them in EXMO?

**Comment by sebastianduesing** [comment]
Yes, that should be fine—just let us know which ontologies you end up requesting terms from (ideally with links to any term request issues you opened on those ontologies' issue trackers), so we'll have a sense for where the terms will eventually be coming from.

**Comment by Exercise-Medicine-Ontology** [comment]
> Hello [@Exercise-Medicine-Ontology](https://github.com/Exercise-Medicine-Ontology)! I've been asked to take over for the next round of review. Thanks for following up with the new version—it's clear that you've put in quite a bit of work to address the issues raised in the previous review. My review is below. I will discuss this with the rest of the OBO Operations Committee at our next call on 2025-05-13. Please feel free to respond with questions, comments, or updates.
> 
> # Criteria
> ## Ontology scope
> * _Was the ontology developed for a very specific purpose or community?_ Yes.
> * _Do the terms fall within the ontology’s stated target domain of knowledge?_ As James said, the core set of terms is reasonably contained within the domain of exercise medicine, and it appears that substantial effort has been made to replace out-of-scope terms with terms from other OBO ontologies. However, there remain a handful of terms with EXMO IRIs that should likely be sourced from other ontologies, as I discuss in more detail below.
> 
> ## Terms with the new ontology prefix
> * _Do the terms follow the OBO identifier scheme?_ Yes. The variance in identifier styles that James identified in the previous version has been corrected, and all IRIs now follow the `http://purl.obolibrary.org/obo/EXMO_0000001` style.
> * _Are there terms with the same meaning available in another OBO Foundry ontology?_ The terms James identified in the prior review have been replaced with imports from the appropriate ontologies. However, there are still a handful of terms that may be able to be replaced with imports, including the following:
>   
>   * `EXMO:0000394` "caring for children" --> `GSSO:003899` "child care"
>   * `EXMO:0000344` "recreational activity" --> `GSSO:008528` "recreation"
>   * `EXMO:0000102` "body composition testing" --> `EFO:0005106` "body composition measurement"
> * _Is there another OBO Foundry ontology whose scope covers any of the new terms?_ Yes. While the number of out-of-scope terms has been substantially reduced since the last review, there are still a few categories of terms that would benefit from being moved to other domain ontologies. In particular, I recommend the following:
>   
>   * Clinical measurement subclasses (e.g.,  `EXMO:0000073` "axillary temperature", `EXMO:0000075` "oral temperature", `EXMO:0000076` "rectal temperature", `EXMO:0000075` "tympanic temperature") should be in CMO.
>   * Injury terms (e.g., `EXMO:0000364` "acute sprain of ankle joint" and `EXMO:0000363` "injury of cruciate ligament of knee") should be in DOID, a controlled vocabulary of diagnosis codes, or similar.
>   * Demographic terms (`EXMO:0000284` "postpartum woman" and `EXMO:0000317` "perimenopausal woman") should be elsewhere. I'm not sure which ontologies would be best for these—perhaps GSSO? Terms like `OBI:0002413` "perimenopausal status" may also be relevant in relocating these terms.
> 
> ## Correct use of imported terms
> * _If the ontology reuses terms from other OBO ontologies, are they used accurately?_ Yes, as far as I can tell. Shawn and James previously noted that it would be good to use ODK for EXMO's imports for long-term stability; I'm not familiar enough with ODK to be certain whether EXMO has taken sufficient action to address Shawn and James' concerns, so I welcome their input on this.
> * _Are imported terms in appropriate hierarchies, and do they preserve the term’s upper-level alignment?_ Yes, EXMO adheres to BFO for its own classes and its imports, and those imports are in appropriate hierarchies. I would recommend one change: currently, the CMO "clinical measurement" branch is imported under "quality". Strictly speaking, this is not in conflict with CMO's modeling, as CMO contains no terms above "clinical measurement", but CMO defines "clinical measurement" as "A quantitative or qualitative value which is the result of an act of assessing a morphological or physiological state or property in a single individual or sample or a group of individuals or samples, based on direct observation or experimental manipulation." Accordingly, it seems that CMO intends "clinical measurement" to be used for _values_ resulting from the process of measuring a characteristic, not the process or the characteristic, and thus it is probably better to import it as an information content entity, perhaps as a subclass of `IAO:0000109` "measurement datum". Clearer modeling in CMO itself would be helpful here.
> * _Are any additional axioms used for these terms correct in both a technical (e.g. passes reasoning) and substantive sense?_ Yes. It appears that the issue with the addition of restrictions to upper-level terms like `BFO:0000019` ("quality") has been fixed by removing those restrictions as James recommended.
> 
> ## Basic review of axiomatic patterns
> * _Are axioms generally stated simply or are they highly complex? (Highly complex axioms will require extra scrutiny.)_ Yes, the axioms are generally quite simple and straightforward.
> * _Are existential restrictions used correctly? (Typical mistakes include “R some (A and B and C)” to mean “(R some A and R some B and R some C)”)._ Yes.
> 
> ## Appropriate use of object properties
> * _Are object properties used in a manner consistent with their definitions, domain, and range? (Examples of incorrect usage include those based on some interpretation of the label of the object property but not actually fitting the property definition or domain and range.)_ Yes. The issue raised by James about the object property `EXMO:0000006` "abbreviation" being out of EXMO's scope has been resolved by removing that term. The issue of nonstandard values for `oboInOwl:hasDbXref` appears to have been resolved by standardizing values into CURIE format as recommended.
> 
> ## Responsiveness to suggested changes
> * Have the developers been willing to fix any identified issues during the review?_ Yes, it is clear that the EXMO team has made a substantial and broadly successful effort to address the issues raised in the previous round of review.
> 
> ## Other
> * It appears that some terms for exercise equipment have axioms relating them to their associated sports or games, while similar terms lack those axioms. For example, `EXMO:0000244` "golf club" has the axiom `'participates in' some 'golf'`, while `EXMO:0000243` "golf ball" has no such axiom. I recommend reviewing the "exercise equipment" branch to ensure that, if you intend to include axioms linking equipment items to sports/games, those axioms are applied consistently across all relevant terms in the branch.
> * As previously noted, ~80 of the ~220 terms with EXMO IDs lack textual definitions. Prioritizing the terms closest to the core of EXMO's domain (e.g., `EXMO:0000350` "anaerobic exercise", `EXMO:0000379` "exercise plan", and similar) for receiving definitions would maximize the payoff of any further work done to add textual definitions.
> 
> # Action Items
> Most of the previous list of action items created by James have been successfully addressed by EXMO. The following list includes the remaining items from that list plus newly-identified items.
> 
> * [ ]  **MUST:** Review terms to identify remaining out-of-scope classes and ensure that out-of-scope classes are imported from other ontologies if equivalents already exist elsewhere OR request them from ontologies with appropriate domains if not.[ ]  **SHOULD:** Import `CMO:0000000` "clinical measurement" as a subclass of `IAO:0000109` "measurement datum" rather than as a subclass of "quality".[ ]  **SHOULD**: Review axioms in general, but particularly in the "exercise equipment" branch, to ensure desired axioms are applied consistently across similar terms.[ ]  **SHOULD**: Add textual definitions, prioritizing terms for core domain concepts, like `EXMO:0000350` "anaerobic exercise".

We updated EXMO: https://github.com/Exercise-Medicine-Ontology/EXMO.
For the out-of-scope classes, we imported them from other ontologies (for "perimenopausal woman", we imported a related concept HP:0040406 "Perimenopausal").
We did not find any OBO ontology incluedes injury (injury is not a disease), so we keep the several related injury classes.
We keep "body composition testing", because it is different from "body composition measurement" (action and value). Other mentioned classes were imported from existing ontologies.
We imported CMO:0000000 "clinical measurement" as a subclass of IAO:0000109 "measurement datum". 
We added axioms for "exercise equipment" classes.
We added textual definitions for most of the core classes.

**Comment by sebastianduesing** [comment]
Thanks for the update. These changes look good to me. I'll raise this to the rest of the OBO operations committee in our next meeting on June 24th for a final decision, and we'll follow up after that.

**Comment by Exercise-Medicine-Ontology** [comment]
> Thank you again for your ontology submission to the OBO Foundry. We are happy to inform you that your ontology EXMO has been accepted following discussion in the OBO Operations Committee meeting on 2025-06-24. Before we can add it to the OBO ontology registry, you need to complete the following steps.
> 
> **Create a metadata record for your ontology to be included in the registry:**
> 
> 1. Create a new file in https://github.com/OBOFoundry/OBOFoundry.github.io/tree/master/ontology, called YOURID.md (there is an “Add file” button in the top right).
> 2. Obtain the already curated metadata that relates to your ontology from https://github.com/OBOFoundry/obo-nor.github.io/blob/master/dashboard-config.yml (you’ll need to scroll down to locate yours; it will have your prefix in the “ - id:” field.)
> 3. Create a pull request to add the metadata record. This pull request should include a link to this issue (the New Ontology Request issue). Here is an example record for the PATO ontology: https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/ontology/pato.md?plain=1
> 
> Your metadata will be reviewed and merged by a member of the OBO Foundry Operations Committee. Permissible content for fields is being documented [here](https://obofoundry.org/faq/permissible-metadata-content.html).
> 
> **To create a PURL registry entry for your ontology:**
> 
> 1. Go to https://github.com/OBOFoundry/purl.obolibrary.org/tree/master/config, click “Add file” and add a file named EXMO.yml.
> 2. Add the desired configuration.
> 3. Make a pull request with a link to this issue. See here for an example of a PURL yml file: https://github.com/OBOFoundry/purl.obolibrary.org/blob/master/config/pato.yml
> 
> Please let us know if you have any questions.

Thank you for accepting EXMO into the OBO Foundry. We are delighted to receive this notification and appreciate your thorough review.

We confirm that all required steps have now been completed. Should any further modifications be needed, please do not hesitate to inform us—we will address them promptly.

**Comment by hoganwr** [comment]
Discussed at OFOC. Awaiting final review and merging of metadata creation PR.

**Comment by nlharris** [comment]
Congratulations, your ontology was accepted! Closing this issue.
